package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@moxy/react-modal",
+  "version": "0.0.0",
+  "description": "A modal, built on top of react-modal, that also fixes the scrollable body in iOS",
+  "author": {
+    "name": "Ivo Lima Silva",
+    "email": "ivolimasilva@outlook.com",
+    "url": "http://ivolimasilva.xyz"
+  },
+  "license": "MIT",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "files": [
+    "lib",
+    "es"
+  ],
+  "scripts": {
+    "build:commonjs": "BABEL_ENV=commonjs babel src -d lib",
+    "build:es": "BABEL_ENV=es babel src -d es",
+    "build": "npm run build:commonjs && npm run build:es",
+    "lint": "eslint --ignore-path .gitignore .",
+    "test": "jest",
+    "prerelease": "npm t && npm run lint && npm run build",
+    "release": "standard-version",
+    "postrelease": "git push --follow-tags origin master && npm publish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moxystudio/react-modal.git"
+  },
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "react-modal": "^3.11.2"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1"
```

Comment:
Why don't you declare `react-dom` as peer dependency as well?

Reply:
I believe I don't actually need to do that since I'm not using `react-dom` directly. Checking other repositories, it does seem that way: [`next-layout`](https://github.com/moxystudio/next-layout/blob/master/package.json#L41), [`wait-for-react`](https://github.com/moxystudio/react-wait-for-react/blob/master/package.json#L54), [`next-intl`](https://github.com/moxystudio/next-intl/blob/master/package.json#L58), ...

I believe I only need to declare it as `peerDependency` if I use like `findDomNode`.

Reply:
Got it 👍